core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -379,7 +379,8 @@ private void tryStart(Instant startedAt) {
                     }
                     reusable = true;
                 } else {
-                    logger().info("Reuse was requested but the environment does not support the reuse of containers");
+                    logger().warn("Reuse was requested but the environment does not support the reuse of containers");
+                    logger().warn("To enable reuse of containers, you must set 'testcontainers.reuse.enable=true' in a file located at ~/.testcontainers.properties");
```

Comment:
Maybe we should make it a bit Windows friendlier? :)

```suggestion
                    logger().warn("To enable reuse of containers, you must set 'testcontainers.reuse.enable=true' in a file located at $HOME/.testcontainers.properties");
```

Reply:
Also, WDYT about merging both messages?

Reply:
sure, how about I change the wording to:
```java
... in a file located at " + System.getProperty("user.home") + "/.testcontainers.properties
```

Reply:
sure 👍

nit: you can use `located at {}/.testcontainers.properties", System.getProperty("user.home")`

Reply:
ok, pushed and ready for review